### ISSUE TITLE
Add a code style guide

### ISSUE THREAD

**Issue by crisisinaptica** [sirjuddington/SLADE#948]
So we all can play nice together.

Or at least a brief general guide or dos and dont's.

**Comment by sirjuddington** [comment]
Yeah it's probably something I should get to doing, the main problem until now has been the fairly inconsistent style (particularly between old and new code). I'm going through and cleaning everything up in master now, so that will no longer be an issue once it's done.

**Comment by crisisinaptica** [comment]
I had not noticed there was a .clang-format file, nice.
